deploy/contents/install/app/install-utils.sh

```diff
@@ -562,6 +562,41 @@ function array_contains_or_empty () {
     return 1
 }
 
+function count_services {
+    parse_env_option "$1"
+    services_count=$((services_count+1))
+    if [ -n "${CP_SERVICES_ENABLED}" ]; then 
+        export CP_SERVICES_ENABLED="${CP_SERVICES_ENABLED},${1}"
+    else 
+        export CP_SERVICES_ENABLED="${1}"
+    fi
+    }
+
+function export_service_all {
+        export CP_INSTALL_SERVICES_ALL=1
+        export CP_SERVICES_ENABLED="all"
+    }
+
+function services_from_pitc {
+    local services_list="$CP_POINT_IN_TIME_CONFIGURATION_DIR/cp-services.json"
+    if [ ! -s "$services_list" ]; then
+       export_service_all
```

Comment:
do we really need this here? 
I believe that we will enable all services if needed https://github.com/epam/cloud-pipeline/pull/3603/files#diff-5c368da199a8aa904a21120fa49deab5adb5dfe6f1f66389a61ec978407c7855R788-R790 here anyway

Reply:
So if file not exist or empty be better to export services_count=0 to enable all services? if allready we have checking of the services_count value